src/lib/drivers/cri.js

```diff
@@ -44,9 +44,13 @@ class CriDriver extends Driver {
         chromeRemoteInterface({port: port, chooseTab: tab}, chrome => {
           this._chrome = chrome;
           this.beginLogging();
-          this.enableRuntimeEvents().then(_ => {
-            resolve();
-          });
+
+          // Switch on events from domains we care about.
+          this.enableRuntimeEvents()
+              .then(_ => this.enableSecurityEvents())
```

Comment:
I think we want to enable this domain within our beforePageLoad gatherer. the events fire immediately, so we want the listener bound already.